lib/ansible/modules/network/asa/asa_command.py

```diff
@@ -68,35 +68,37 @@
 """
 
 EXAMPLES = """
-# Note: examples below use the following provider dict to handle
+# Note: examples below use the following group_vars/asa.yaml file to handle
 #       transport and authentication to the node.
 ---
-vars:
```

Comment:
Do you still need this line so Ansible loads lines 74-80 as variables?

Reply:
This is where I'm not sure if I'm explaining it correctly.  In my test, I have a group_vars/ directory with an asa.yml file that contains that section (lines 74-80).  Then I have a playbook that includes all the examples (starting with line 83).   Do we want to encourage the use of group_vars/ or should I go back to having this all in the playbook itself?

Reply:
based on sidebar discussion, removing the variables altogether and just keeping these as examples, as most modules use this approach